packages/clerk-js/src/utils/querystring.ts

```diff
@@ -6,6 +6,6 @@ export const getQueryParams = (queryString: string) => {
   });
 };
 
-export const stringifyQueryParams = (params: {}) => {
+export const stringifyQueryParams = (params: any) => {
```

Comment:
Even though `qs.stringify` is typed as `function stringify(obj: any, options?: IStringifyOptions): string;`,  I think we should choose a slightly stricter type here, probably something along the lines of `Record<string, unknown> | Array<unknown>`. 

We can always revise in the future if needed.

Reply:
We actually don't use `stringifyQueryParams` anywhere so it should be fine to change this